app/build.gradle

```diff
@@ -8,7 +8,7 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'jacoco'
 apply plugin: 'androidx.navigation.safeargs.kotlin'
 apply plugin: 'com.google.android.gms.oss-licenses-plugin'
-
+apply plugin: 'com.google.firebase.crashlytics'
```

Comment:
What does the plugin do?

I ran into this as well when setting up FCM and after inspection, we realized that this plugin was handling the crashlytics initialization which we were better equipped to do ourselves. This may not be the case here, but might be worth checking out if you haven't already. 🙂